WireUI/Sources/WireIndividualToTeamMigrationUI/Components/PageContainer.swift

```diff
@@ -40,20 +40,25 @@ struct PageContainer<Content: View>: View {
     }
 
     var body: some View {
-        ScrollView {
-            VStack {
-                Text(String.formated(key: "individualToTeam.progressCount", bundle: .module, step, stepCount))
-                    .wireTextStyle(.subline1)
-                    .foregroundStyle(Color(uiColor: ColorTheme.Base.secondaryText))
-                Spacer()
-                    .frame(height: 12)
-                Text(stepTitle)
-                    .wireTextStyle(.h2)
-                Spacer(minLength: 36)
-                content
+        GeometryReader { proxy in
+            ScrollView {
+                VStack {
+                    Text(String.formated(key: "individualToTeam.progressCount", bundle: .module, step, stepCount))
+                        .wireTextStyle(.subline1)
+                        .foregroundStyle(Color(uiColor: ColorTheme.Base.secondaryText))
+                    Spacer()
+                        .frame(height: 12)
+                    Text(stepTitle)
+                        .wireTextStyle(.h2)
+                    Spacer(minLength: 36)
+                    content
+                }
+                .frame(
+                    height: proxy.size.height - 24
+                )
             }
+            .padding(.horizontal, 16)
+            .padding(.bottom, 24)
         }
-        .padding(.horizontal, 16)
-        .padding(.bottom, 24)
```

Comment:
I wonder if we could get away without `GeometryReader`.
Another point I find weird is that the whole content of a screen is put into a scroll view.
Looking at the design
<img width="282" alt="Screenshot 2024-12-12 at 13 15 07" src="https://github.com/user-attachments/assets/f9fd77ff-d92b-4ea0-b392-91291fdb8bda" />
I'd have the bottom button outside the scroll view.

Reply:
Without the GeometryReader, hugs its content, and the button is in the middle of the screen. This would be solved by a floating button.

The issue with a floating button would be this case:

![Screenshot 2024-12-12 at 15 47 47](https://github.com/user-attachments/assets/c3dd887b-789b-470b-adec-3082e6035ec6)

The user would have an inactive floating button, greyed out and disabled, with no clue on how to continue.